src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -108,6 +108,11 @@ internal static IEnumerable ReadAsEnumerable(object value)
         {
             IEnumerable values = null;
 
+            if (value is JArray jArray)
+            {
+                return jArray;
+            }
```

Comment:
Is this change not in /dev?  Did we miss a port?

Reply:
It is not in dev - basically, the workers sometimes JsonConvert to a JArray, which already implements IEnumerable. I can backport it desired, though there's nothing in dev that exercises this particular codepath